quran/models.py

```diff
@@ -1,19 +1,88 @@
 '''Models describing abstractions used at Quran app'''
-from django.db import migrations, models
+from os import path
```

Comment:
Please, don't do it, `os.path` is short enough.
```suggestion
import os
```

Reply:
OK